### ISSUE TITLE
DEV-20 Pie Chart 3d: does not work with legend

### ISSUE THREAD

**Issue by KovDimaY** [KovDimaY/React-Highcharts#38]
There is a strange bug, that appears when the 3d version of a Pie Chart is interacting with the legend (enable and disable series).
You can see the bug on the gif attached:
![issue dev-20](https://user-images.githubusercontent.com/26466644/33667517-56e07ff6-da9d-11e7-9d5f-2e4a32714911.gif)

This issue appears both in donat mode and usual 3d, it appears with data labels and without. The only pattern that produces this bug is that the chart is in 3d. For 2D it works ok:
![issue dev-20-2d](https://user-images.githubusercontent.com/26466644/33667615-a3501900-da9d-11e7-80af-856154227101.gif)

I think this error is from Highcharts or from the React wrapper for the Highcharts because the actions that lead to the error are not connected with the functionality of the project.

Any help or ideas are welcomed! :)

**Comment by KovDimaY** [comment]
Today I have tried it with native Highchart of the last version and it works without this error (still not perfect, but at least the error happens not in 100% of the cases). So maybe it makes sense to update the library and it will solve the problem (if React Highchart also updates with the native library).

**Comment by KovDimaY** [comment]
@TetianaKlymchuk could you take a look at this issue, please?

**Comment by TetianaKlymchuk** [comment]
Yes, let me try... 👍

**Comment by TetianaKlymchuk** [comment]
Yes, @KovDimaY, you were absolutely right! The problem was on the package side, I updated the version and the bug disappeared:

![pie](https://user-images.githubusercontent.com/28005338/46766495-70af1e00-cce2-11e8-8ba8-aab4c8c3b34b.gif)
Also I cleaned a bit the dependencies deleting the ones that we do not use in the project.

I will create a PR in several minutes! 💃
